sci-electronics/librepcb/librepcb-0.1.0_p201805.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit qmake-utils
+
+DESCRIPTION="Free EDA software to develop printed circuit boards"
+HOMEPAGE="http://librepcb.org/"
+MY_P="LibrePCB-first_pcb"
+SRC_URI="https://github.com/LibrePCB/LibrePCB/archive/first_pcb.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+S="${WORKDIR}/${MY_P}"
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5[ssl]
+	dev-qt/qtopengl:5
+	dev-qt/qtprintsupport:5
+	dev-qt/qtsql:5
+	dev-qt/qtwidgets:5
+	dev-qt/qtwebkit:5
+	dev-qt/qtxml:5
+	sys-libs/zlib"
+
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	sed -i '/Version/d' packaging/unix/librepcb.desktop || die
+	eapply_user
```

Comment:
Probably can be replaced with default.

Reply:
`s/can/should`

Reply:
also please fix the `Version` thin the desktop file with a patch, and submit the patch upstream

Reply:
@SoapGentoo It's already fixed [here](https://github.com/LibrePCB/LibrePCB/pull/176)